opm/input/eclipse/Schedule/Well/Connection.hpp

```diff
@@ -250,6 +308,8 @@ namespace RestartIO {
         // Whether or not this Connection is subject to WELPI scaling.
         bool m_subject_to_welpi = false;
 
+        FilterCake m_filter_cake;
+
```

Comment:
Does every connection always have a filter cake model?  If not, this might be better as a `std::optional<FilterCake>` instead.